waku/v2/node/wakunode2.nim

```diff
@@ -409,30 +409,61 @@ proc mountStore*(node: WakuNode, store: MessageStore = nil, persistMessages: boo
 
 when defined(rln):
   proc mountRlnRelay*(node: WakuNode,
-                      ethClientAddress: Option[string] = none(string),
-                      ethAccountAddress: Option[Address] = none(Address),
-                      membershipContractAddress:  Option[Address] = none(Address)) {.async.} =
+                      ethClientAddrOpt: Option[string] = none(string),
+                      ethAccAddrOpt: Option[Address] = none(Address),
+                      memContractAddOpt:  Option[Address] = none(Address),
+                      groupOpt: Option[seq[IDCommitment]] = none(seq[IDCommitment]),
+                      memKeyPairOpt: Option[MembershipKeyPair] = none(MembershipKeyPair),
+                      memIndexOpt: Option[uint] = none(uint)) {.async.} =
     # TODO return a bool value to indicate the success of the call
     # check whether inputs are provided
-    doAssert(ethClientAddress.isSome())
-    doAssert(ethAccountAddress.isSome())
-    doAssert(membershipContractAddress.isSome())
+    doAssert(ethClientAddrOpt.isSome())
+    doAssert(ethAccAddrOpt.isSome())
+    doAssert(memContractAddOpt.isSome())
+    doAssert(groupOpt.isSome())    # TODO this check is not necessary for a dynamic group
+    doAssert(memKeyPairOpt.isSome())
+    doAssert(memIndexOpt.isSome())
+
+    let 
```

Comment:
Maybe we could have a brief comment on what each of these things correspond to? Right now not very clear, at least to me

E.g. ID key correspond to private key that can... ID commitment public key identity (?) ...